pkg/webhooks/admission/jobflows/validate/admit_jobflow.go

```diff
@@ -0,0 +1,118 @@
+package validate
+
+import (
+	"fmt"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	whv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/klog"
+	jobflowv1alpha1 "volcano.sh/apis/pkg/apis/flow/v1alpha1"
+	"volcano.sh/apis/pkg/apis/helpers"
+	"volcano.sh/volcano/pkg/webhooks/router"
+	"volcano.sh/volcano/pkg/webhooks/schema"
+	"volcano.sh/volcano/pkg/webhooks/util"
+)
+
+func init() {
+	router.RegisterAdmission(service)
```

Comment:
the `defaultEnabledAdmission` in `cmd/webhook-manager/app/options/options.go` should also be updated.